stable/database/values.yaml

```diff
@@ -54,6 +54,10 @@ admin:
   # tlsCACert:
   #   secret: nuodb-ca-cert
   #   key: ca.cert
+  # tlsKeyStore:
+  #   secret: nuodb-server-keystore
+  #   key: server.p12
```

Comment:
Is `server.p12` going to work? We always mount the secret in nuoadmin.p12.

Reply:
its the same text as in the admin charts